### PR TITLE
Modify Padding navbar menu

### PR DESCRIPTION
**Pull request by ElishaSamPeterPrabhu**
## Description
Padding is removed for drop down menu in navbar

References # #1841

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation update

## How Has This Been Tested?
UI changes are dev tested and compared with figma

## Checklist

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have checked my code and corrected any misspellings

### CODE REVIEW

File: stencil-workspace/src/components/modus-navbar/button-menu/modus-navbar-button-menu.scss [hunk 1]
```diff
@@ -5,6 +5,7 @@
 
   background-color: $menu-background-color;
   cursor: default;
+  padding: 0;
```
Comment (msankaran0712) [inline stencil-workspace/src/components/modus-navbar/button-menu/modus-navbar-button-menu.scss:8]: Padding needs to be removed on all the pop-ups in Navbar, so the fix should be made here 
image

File: stencil-workspace/src/components/modus-navbar/modus-navbar.vars.scss [hunk 2]
```diff
@@ -8,7 +8,7 @@
   display: flex;
   flex-direction: column;
   max-width: 384px;
-  padding: $rem-8px;
+  padding: 0px;
```
Comment (coliff) [inline stencil-workspace/src/components/modus-navbar/modus-navbar.vars.scss:11]: be sure to lint the CSS. zero pixel values should not display the unit (REF: https://stylelint.io/user-guide/rules/length-zero-no-unit/)

**Comment by ElishaSamPeterPrabhu** [comment]
> @ElishaSamPeterPrabhu Are the open comments in this PR resolved?

Yes

File: stencil-workspace/src/components/modus-dropdown/modus-dropdown.scss [hunk 3]
```diff
@@ -7,7 +9,9 @@
   }
 
   .dropdown-list {
-    border-radius: $rem-2px;
+    border: 1px solid $modus-dropdown-list-border-color;
```
Comment (msankaran0712) [inline stencil-workspace/src/components/modus-dropdown/modus-dropdown.scss:12]: On dark theme, border color is not correct

File: stencil-workspace/storybook/stories/components/modus-dropdown/modus-dropdown-storybook-docs.mdx [hunk 4]
```diff
@@ -18,19 +18,19 @@
     Dropdown
   </modus-button>
   <modus-list slot="dropdownList">
-    <modus-list-item size="condensed">Item 1</modus-list-item>
-    <modus-list-item size="condensed">Item 2</modus-list-item>
-    <modus-list-item size="condensed">Item 3</modus-list-item>
+    <modus-list-item size="condensed" borderless>Item 1</modus-list-item>
```
[the remark this answers is not in the document]
Comment (ElishaSamPeterPrabhu) [reply stencil-workspace/storybook/stories/components/modus-dropdown/modus-dropdown-storybook-docs.mdx:21]: Added lines from stories

File: stencil-workspace/src/global/themes.scss [hunk 5]
```diff
@@ -411,6 +411,9 @@
   // Autocomplete
   --modus-autocomplete-no-results-message-color: var(--modus-gray-7);
   --modus-autocomplete-no-results-subtext-color: var(--modus-gray-4);
+
+  // Dropdown
+  --modus-dropdown-list-border-color: #cfcfd2;
```
Comment (msankaran0712) [inline stencil-workspace/src/global/themes.scss:416]: Where did we get this hex code from?

**Review by msankaran0712 — APPROVED** [review]

**Pull request merged**